docs/usage.md

```diff
@@ -49,13 +58,12 @@ It is recommended to limit the Nextflow Java virtual machines memory. We recomme
 NXF_OPTS='-Xms1g -Xmx4g'
 ```
 
-<!-- TODO nf-core: Document required command line parameters to run the pipeline-->
-
 ## Running the pipeline
+
 The typical command for running the pipeline is as follows:
 
 ```bash
-nextflow run nf-core/scrnaseq --reads '*_R{1,2}.fastq.gz' -profile docker
+nextflow run nf-core/scrnaseq --reads '*_R{1,2}.fastq.gz' --type '10x' --aligner 'kallisto' --barcode_whitelist 10xv2.whitelist.txt --fasta human.fasta --gtf human.gtf -profile docker
```

Comment:
Do we really need quotes around `kallisto`?
Typical command is run with Kallisto then I suppose?
Should it not be specified by default in the config files? (with the specific files)